frontend/src/pages/ChatPage.vue

```diff
@@ -1,52 +1,52 @@
 <template>
   <v-container>
-    <h1>Chat Page, Room Title</h1>
+    <h1>{{ roomInfo.title }}</h1>
     <v-row>
       <v-col cols="12">
-        <li v-for="message in messages" :key="message.title">
+        <li v-for="message in messages" :key="message.title" class="pb-1">
           <v-card
             outlined
           >
-            <v-list-item three-line style="text-align: left;">
+            <v-list-item single-line style="text-align: left;">
               <v-list-item-content>
-                <v-list-item-title >{{ message.title }}</v-list-item-title>
```

Comment:
[IMO]
Shouldn't we display something on title?(ex. date, message index)

Reply:
I agree. It will implement in another PR.